assets/sass/components/buttons.scss

```diff
@@ -249,6 +249,22 @@ $btnConf: (
     }
 }
 
+
+
+.btn--next {
+    background-image: arrow-btn();
```

Comment:
Thinking about it, maybe css clip-paths are the way to go here. e.g. http://bennettfeely.com/clippy/
That's the way all the wonky stuff on https://membership.theguardian.com/ was done.

Reply:
Just looking into this. Here's the best I could manage:
![image](https://user-images.githubusercontent.com/394376/37827512-b478cbc8-2e8f-11e8-9a96-f17b29015267.png)

(using `clip-path: polygon(0% 0%, 80% 0%, 95% 50%, 80% 100%, 0% 100%)`)

The original one, though, has a rounded edge on the arrow:

![image](https://user-images.githubusercontent.com/394376/37827535-c7ad25a4-2e8f-11e8-90b7-fedffb73082e.png)

[Apparently](https://stackoverflow.com/questions/31765345/how-to-round-out-corners-when-using-css-clip-path), though, you can't easily round corners using CSS clip-path (other than literally draw each point in the corner). Not sure how much of a compromise we can make here.